File: footballRepository/src/main/java/com/turk/footballrepository/dataSource/local/AppLocalDatabase.java
package com.turk.footballrepository.dataSource.local;

import androidx.room.Database;
import androidx.room.RoomDatabase;
import com.turk.dtos.footballmatch.FootBallMatch;
import com.turk.footballrepository.dataSource.local.source.FootBallMatchDao;

@Database(entities = {FootBallMatch.class,}, version = 1, exportSchema = false)

public abstract class AppLocalDatabase extends RoomDatabase {


    public abstract FootBallMatchDao getFootBallMatchDao();

    

}

File: footballRepository/src/main/java/com/turk/footballrepository/dataSource/local/AppLocalDatabase.kt
package com.turk.footballrepository.dataSource.local


import androidx.room.Database
import androidx.room.RoomDatabase
import com.turk.dtos.serverObjects.FootballMatchResponse
import com.turk.footballrepository.dataSource.local.source.FootBallMatchDao

@Database(entities = [FootballMatchResponse::class], version = 1, exportSchema = false)
abstract class AppLocalDatabase : RoomDatabase() {
    abstract val footBallMatchDao: FootBallMatchDao?
}